sys-libs/pam_wrapper/pam_wrapper-1.1.3-r1.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} )
+CMAKE_ECLASS=cmake
+inherit cmake-multilib python-r1
+
+DESCRIPTION="A tool to test PAM applications and PAM modules"
+HOMEPAGE="https://cwrap.org/pam_wrapper.html"
+SRC_URI="https://www.samba.org/ftp/pub/cwrap/${P}.tar.gz
+	https://ftp.samba.org/pub/cwrap/${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="test"
+
+RESTRICT="!test? ( test )"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="${PYTHON_DEPS}
+	sys-libs/pam:0=[${MULTILIB_USEDEP}]
+"
+
+DEPEND="${RDEPEND}
+	test? ( dev-util/cmocka[${MULTILIB_USEDEP}] )
+"
+
+multilib_src_configure() {
```

Comment:
This probably isn't enough - we likely need to call this for each Python implementation & build it

Reply:
See e.g. [this](https://gitweb.gentoo.org/repo/gentoo.git/tree/sci-geosciences/mapserver/mapserver-7.6.2-r1.ebuild#n177)

Reply:
I don't understand how to do it as it depends on CmakeList.txt and tests/CmakeList.txt

Reply:
In multilib_src_configure, you need to wrap everything in a function like:
```
    build_for_python() {
        # local mycmakeargs=()
        # cmake_src_configure
    }
```

then in multilib_src_configure:
```
python_foreach_impl build_for_python
```

Something like that will work. You'll need to do the same in compile + install -- the example I linked is really similar so should help